fuse_core/include/fuse_core/macros.h

```diff
@@ -58,7 +58,7 @@
 #include <memory>
 #include <string>
 
-// Required by make_aligned_shared, that uses Eigen::aligned_allocator<T>().
+// Required by __MAKE_SHARED_ALIGNED_DEFINITION, that uses Eigen::aligned_allocator<T>().
```

Comment:
Simply updating the comment, because we finally don't use `make_aligned_shared`, which doesn't exist.